spv/envelope.go

```diff
@@ -36,3 +38,72 @@ func (e *Envelope) ParentTx(txID string) (*bt.Tx, error) {
 	}
 	return bt.NewTxFromString(env.RawTx)
 }
+
+// Bytes takes an spvEnvelope struct and returns the serialised binary format.
+func (e *Envelope) Bytes() ([]byte, error) {
+	flake := make([]byte, 0)
```

Comment:
Flake is a bit of a funky name for this IMO, when I first read this I had no idea what it was for/what it meant, and then realised after reading further that it's the byte slice to be returned.

Internally, we typically just call this something like `bytes`, or `buffer`, such as here for example: https://github.com/libsv/go-bc/blob/master/block.go#L33

Normally I wouldn't care, but given we're trying to have this also be a reference implementation, could this be renamed to something clear, such as `bytes`, before you merge?